rest/http_rest_endpoint.cc

```diff
@@ -204,12 +204,48 @@ sendResponse(int code,
              std::string body, std::string contentType,
              RestParams headers)
 {
-    for (auto & h: endpoint->extraHeaders)
-        headers.push_back(h);
```

Comment:
If we are in-lining the code to avoid this push_back loop then I would be tempted to use boost::join instead which should avoid all copies or simply modify the HttpResponse to take two sets of headers instead.